frame/collective/src/lib.rs

```diff
@@ -591,8 +488,11 @@ decl_module! {
 		///   - 1 storage mutation `Voting` (codec `O(M)`)
 		/// - 1 event
 		/// # </weight>
+		// NOTE: surprising difference between old manual weight and new generated weight:
+		// (49155000 as Weight)
+		//	.saturating_add((295000 as Weight).saturating_mul(m as Weight))
```

Comment:
| before | after |
|------|-----|
| 30_000_000 + 500_000 * m | 46_165_000 + 264_000*m |

surprising but not too bad seems like the analysis moved some cost related to m to the constant cost 🤔